instance-billing-flavor-check.spec

```diff
@@ -0,0 +1,39 @@
+Summary:   Cloud Billing Flavour Check
+Name:      instance-billing-type-check
+Version:   %%VERSION
+Release:   0
+License:   GPL3.0-only
+Group:     Productivity/Networking/Web/Servers
```

Comment:
This is not a server, it's a client tool

Reply:
I did take a look at other Python applications, 
if that `Group` is not OK, I'm not sure what it should be there
for that, I took a look at [cloud-regionsrv-client spec](https://github.com/SUSE-Enceladus/cloud-regionsrv-client/blob/master/cloud-regionsrv-client.spec#L25)

Reply:
is `System/Management` correct for this case (as other client applications have that) ?